realign goals, questions, and content (#389)
carpentries-incubator/hpc-intro: _episodes/17-resources.md

---
title: "Using resources effectively"
teaching: 10
exercises: 30
exercises: 20
questions:
- "How do we monitor our jobs?"

> - "How can I get my jobs scheduled more easily?"

Reviewer: May want to leave this. On many systems there are different queues - even on a homogeneous system, there is usually a production and debug queue.  Perhaps ask participants to see how many queues are available and discuss their possible purposes.

Author: We can't leave these questions in place, as the existing lesson does not answer them. Discussion of available queues is a good idea, but out of scope for this PR.

- "Look up job statistics."

> - "Use past performance to make more accurate resource requests in job scripts."

Reviewer 2: ```suggestion
- "Make more accurate resource requests in job scripts based on data describing past performance."
```

Author: Wordy, but more meaningful.
